### PR TITLE
Added endpoints to WebApi layer

### PR DESCRIPTION
**Pull request by salvanicola**
- implement controller for code request and authentication request
- mapper from Result errors to HttpResponseExceptions

### CODE REVIEW

File: src/Kerbero.Common/Errors/CommonErrors/InvalidParametersError.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ public class InvalidParametersError: KerberoError
 {
 	public InvalidParametersError(object parameter,
 		string? message = "There are missing or wrong parameter in the request: ") : base(
-		message + parameter.ToString())
+		message + parameter + ".")
```
Comment (kuamanet) [inline src/Kerbero.Common/Errors/CommonErrors/InvalidParametersError.cs:9]: Why? Do we expect everyone that builds an `InvalidParametersError` to know that a `.` will be appended to their message?

File: src/Kerbero.WebApi/ConfigureService.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using Kerbero.Common.Models;
+
+namespace Kerbero.WebApi;
+
+public static 
+	class ConfigureService
+{
+	public static IServiceCollection AddWebApiServices(this IServiceCollection services)
+	{
+		services.AddScoped<Interactor<string, Uri>, ProvideNukiAuthRedirectUrlInteractor>();
```
Comment (kuamanet) [inline src/Kerbero.WebApi/ConfigureService.cs:12]: Wrap `string` and `Uri` with classes scoped to the interactor context

File: tests/Kerbero.WebApi.Tests/NukiAuthenticationControllerTest.cs [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+	{
+		_interactorToken = new Mock<InteractorAsync<NukiAccountExternalRequestDto, NukiAccountPresentationDto>>();
+		_interactorCode = new Mock<Interactor<string, Uri>>();
+		_controller = new NukiAuthenticationController(_interactorCode.Object, _interactorToken.Object);
+	}
+	
+	[Fact]
+	public void RedirectForCode_AndRedirect_Test()
+	{
+		// Arrange
```
Comment (kuamanet) [inline tests/Kerbero.WebApi.Tests/NukiAuthenticationControllerTest.cs:37]:
```suggestion
```

**Review by kuamanet — CHANGES_REQUESTED** [review]

File: src/Kerbero.Common/Errors/CommonErrors/InvalidParametersError.cs (hunk 1 above)
Comment (salvanicola) [reply src/Kerbero.Common/Errors/CommonErrors/InvalidParametersError.cs:9]: and how i can hide it?

**Review by kuamanet — APPROVED** [review]

**Pull request merged**